server/api/crud/model_monitoring/deployment.py

```diff
@@ -1115,15 +1123,15 @@ def set_credentials(
                 and not endpoint_store_connection.startswith("mysql")
                 and not endpoint_store_connection.startswith("sqlite")
             ):
-                raise mlrun.errors.MLRunInvalidArgumentError(
+                raise mlrun.errors.MLRunInvalidMMStoreType(
                     "Currently only MySQL/SQLite connections are supported for non-v3io endpoint store,"
                     "please provide a full URL (e.g. mysql+pymysql://<username>:<password>@<host>:<port>/<db_name>)"
                 )
             secrets_dict[
                 mlrun.common.schemas.model_monitoring.ProjectSecretKeys.ENDPOINT_STORE_CONNECTION
             ] = endpoint_store_connection
         else:
-            raise mlrun.errors.MLRunInvalidArgumentError(
```

Comment:
just a recommendation you can replace lines L1121-L1125
`            if (
                endpoint_store_connection != "v3io"
                and not endpoint_store_connection.startswith("mysql")
                and not endpoint_store_connection.startswith("sqlite")
            ):`

with a simple 

`if not endpoint_store_connection.startswith(("v3io", "mysql", "sqlite")):`

Moreover we should consider having a list that includes all of the current supported store schemas.